doc/nrf/known_issues.rst

```diff
@@ -1895,7 +1896,7 @@ NCSDK-18422: Serial recovery fails to write to slots in QSPI
 NCSDK-18108: ``s1`` variant image configuration mismatch
```

Comment:
NCSDK-18108 is fixed

Reply:
Yes, the tag update referred to NCSDK-11308. (The tag always precedes the known issue, not follows it.) Is NCSDK-11308 also fixed, @osaether ?

Reply:
@greg-fer Sorry about that. 11308 is not fixed. There is a PR with a fix but it is not marked with NCS 2.3 milestone.